FlowCal/io.py

```diff
@@ -1527,8 +1527,20 @@ def __new__(cls, infile):
         detector_voltage = tuple(detector_voltage)
 
         # Amplifier gain: Stored in the keyword parameter $PnG for channel n.
-        amplifier_gain = [fcs_file.text.get('$P{}G'.format(i))
-                          for i in range(1, num_channels + 1)]
+        amplifier_gain = []
+        for i in range(1, num_channels + 1):
+            channel_amp_gain = fcs_file.text.get('$P{}G'.format(i))
+
+            # The FlowJo Collector's Edition version 7.5.110.7 software saves
+            # the amplifier gain in keyword parameters CytekP01G, CytekP02G,
+            # CytekP03G, ... for channels 1, 2, 3, ...
+            if channel_amp_gain is None and 'CREATOR' in fcs_file.text and \
+                    'FlowJoCollectorsEdition' in fcs_file.text.get('CREATOR'):
+                try:
+                    channel_amp_gain = fcs_file.text['CytekP{:02d}G'.format(i)]
```

Comment:
Hmmm, that's a good question. Hadn't thought about that. Does that mean we can just get rid of the `try-except` entirely then? By `pass`ing in the `try-except`, it preserves the previous value of `channel_amp_gain` upon `KeyError`, but we should only have gotten to that `try-except` if `channel_amp_gain` was `None`. `.get()` should default to `None` if the key doesn't exist, which sounds like the same behavior to me. If you agree, I can go update that (and make sure I get the same unit test results, etc. etc.).